packages/shorebird_cli/lib/src/auth/auth.dart

```diff
@@ -170,19 +180,15 @@ class Auth {
 }
 
 extension on oauth2.AccessCredentials {
-  String get email {
+  String? get email {
```

Comment:
I don't have the context to know, but should be be obscuring the fact that this is an email here? e.g. just treating it like an opaque identifier?

Reply:
This is currently used to extract the email from the claims so that we can include the user's email in the output of commands like `shorebird account` or `shorebird login` (when they're already logged in). This code will have to be refactored once we support multiple OAuth providers 👍